pkg/utils/feed.go

```diff
@@ -41,6 +41,42 @@ const (
 	feedPatchPath = "/spec/feeds"
 )
 
+// Define resource kind.
```

Comment:
What are these constants used for?

Reply:
> What are these constants used for?
They are used for kind judgment in AggregateFeedsStatus.

Reply:
No references for these constants. They're not in use. Please remove them.